migrations/003-create-table-blog-posts.ts

```diff
@@ -36,6 +36,20 @@ const blogPosts = [
     is_published: true,
     user_id: 2,
   },
+  {
+    title: "Special Blogpost 2",
```

Comment:
let's describe what is special about both blog posts in both the

- title
- content

I'll take this over

Reply:
Done in da3dbe4